python/src/aiconfig/scripts/aiconfig_cli.py

```diff
@@ -33,7 +37,7 @@ async def main_with_args(argv: list[str]) -> int:
 
 def run_subcommand(argv: list[str]) -> Result[str, str]:
     LOGGER.info("Running subcommand")
-    subparser_record_types = {"edit": EditServerConfig}
+    subparser_record_types = {"edit": EditServerConfig, "rage": rage.RageConfig}
```

Comment:
Q: why is this called `subparser_...`?

Reply:
Each one defines the struct that a particular subcommand corresponds to. Each subcommand gets one struct, and that subcommand then automatically gets arguments corresponding to the fields in that struct, which are then automatically populated.

Reply:
Sorry just to clarify, what are subcommands, how do we anticipate client to pass this into to server?